spec/rspec/mocks/any_instance_spec.rb

```diff
@@ -123,6 +123,11 @@ def private_method; :private_method_return_value; end
               expect(klass.new.one.plus(1)).to eq(2)
               expect(klass.new.one.plus(2)).to eq(3)
             end
+
+            it 'can use a chain of methods to perform an expectation' do
```

Comment:
Given this is the `any_instance_spec.rb` file, isn't the entire file about demonstrating that `expect_any_instance_of`?  I thought that what's unique about this spec example vs the others in this file is that it uses `receive_message_chain(...).with(...)` so it seems like the doc string should call that out.